infra/cf-state-machine-role.yaml

```diff
@@ -25,24 +25,6 @@ Resources:
             Action: "sts:AssumeRole"
       Path: "/"
       Policies:
-        - PolicyName: !Sub "can-trace-xray-${Environment}"
-          PolicyDocument:
-            Version: "2012-10-17"
-            Statement:
-              - Effect: Allow
-                Action:
-                  - "xray:PutTraceSegments"
-                  - "xray:PutTelemetryRecords"
-                  - "dynamodb:DescribeTable"
-                  - "xray:GetSamplingRules*"
-                  - "xray:GetSamplingTargets"
```

Comment:
why do we not need these xray permissions anymore?

Reply:
we do need them, I added them back